CHANGELOG.md

```diff
@@ -7,11 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
-## [1.15.3] - 2020-08-14
+## [1.16.0] - 2020-09-10
 
-### Fixed
+### Changed
 
-- issue where internal header where font wasn't being set
+- `hmrcTimeoutDialog` Allow users to set a different URL for timeouts than manual sign outs
```

Comment:
This seems to be replacing the previous release rather than adding a new one, can you 

```
## [Unreleased]

## [1.16.0] - 2020-09-10

### Changed

- `hmrcTimeoutDialog` Allow users to set a different URL for timeouts than manual sign outs

## [1.15.3] - 2020-08-14

### Fixed

- issue where internal header where font wasn't being set

```

Reply:
Oops! Should be fixed now.